drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -172,12 +172,12 @@ void at86rf2xx_set_page(at86rf2xx_t *dev, uint8_t page)
 
 uint16_t at86rf2xx_get_pan(at86rf2xx_t *dev)
 {
-    return dev->pan;
+    return byteorder_ntohs(dev->pan);
 }
 
 void at86rf2xx_set_pan(at86rf2xx_t *dev, uint16_t pan)
 {
-    dev->pan = pan;
+    dev->pan = byteorder_htons(pan);
```

Comment:
This is the culprit of my previous reception problems, btw. If I assess the situation correctly the byteorder of the PAN is returned wrong on `driver->get` on most devices, it should be in network byte order but is in little-endian (this PR intentionally doesn't fix that, since it is not topic of this PR).
@haukepetersen @jfischer-phytec-iot @jremmert-phytec-iot @thomaseichinger can you assess the situation for kw2xrf and xbee?